### PR TITLE
DE-1149 Renew Code Challenge

### PR DESCRIPTION
**Pull request by beacover**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,70 +6,51 @@ successfully complete these steps.
...
-  * You can create a Dockerfile with the enviroment and just write in a txt how you will set up a cron job.
+  * You can create a Dockerfile with the environment and just write in a txt how you will set up a cron job.
   * Add another document in the `working_folder` explaining how you did it and
   some evidences.
-7. [OPTIONAL] Try yo load the CSV files donwload from the api using an event processing platform. You can choose the 
+8. [OPTIONAL] Try to explain how would you ingest data from the api using an event processing platform. You can choose the 
    event platform you want Kafka, RabbitMQ, Celery... etc. There is not need to create the full architecture code, but
    otherwise create a file called streaming_events.txt explaining us how will you build this architecture.
-8. [OPTIONAL] Create the enviroment to deploy your API in to a production enviroment.
+9. [OPTIONAL] Create the environment to deploy your API in to a production environment.
```
Comment (antonzhyltsou-cw) [inline README.md:53]:
```suggestion
9. [OPTIONAL] Create the deployment of your API.
```

**Review by antonzhyltsou-cw — CHANGES_REQUESTED** [review]
Some things to change.

**Review by antonzhyltsou-cw — APPROVED** [review]
LGTM.

File: README.md [hunk 2]
```diff
@@ -6,70 +6,51 @@ successfully complete these steps.
...
-6. [OPTIONAL] Following the Serverless approach, put this pipeline to automatically
+or programming language you want, there is no restriction. The API should have a persistence in a db.
+6. In the `working_folder` create a directory call `querys`. You will find a csv file called `weather_forecasts_history.csv` with the following fields:
+    * `City`: name of the city
+    * `Created`: date when the forecast on the Applicable Date is made
+    * `Applicable Date`: date when the forecast takes place
+    * `Wind Speed`: measure of the speed of the wind
+    * `Temperature`: measure of the temperature  
+            
+   Load this data into your db and create a file called `query.txt` in the `querys` directory with the SQL query code to get a report of:
```
[the remark this answers is not in the document]
Comment (beacover) [reply README.md:40]: Done

**Pull request merged**